UPGRADING.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading Replicated Security
+
+This guide provides instructions for upgrading to specific versions of Replicated Security.
+
+## [v3.0.x](https://github.com/cosmos/interchain-security/releases/tag/v3.0.0-rc0)
```

Comment:
Shouldn't this link to https://github.com/cosmos/interchain-security/tree/release/v3.0.x?

Reply:
Maybe change the text from `v3.0.x` to `v3.0.0` then, but not a biggie